docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
+
+## Prerequisites
+
+To use the **Slack connector**, a Slack app must be registered with the Slack workspace you would like to send messages to, and a respective OAuth token needs to be configured as secret in your cluster. Follow [these steps in the appendix](#appendix) to learn how to set this up.
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, either change the type of an existing task using the **change type** context menu, or create a new connector task by using the **append** context menu. Follow [our guide on using connectors](../use-connectors.md) to learn more.
+
+## Make your Slack connector executable
+
+To make the **Slack connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![slack connector red properties](../img/connectors-slack-red-properties.png)
+
+1. Set **Authentication** to Slack OAuth token, which is stored as secret. For example `secrets.SLACK_OAUTH_TOKEN`
+2. Set **Channel/User Name** to either a **user** or a **channel** to send the message to
+   1. A **channel** is specified by an unique identifier starting with an `#` (example: `#myChannel`)
+   2. A **user** is specified by its user name starting with an `@` (example: `@myUser`)
+3. Set **Message** to the message string you would like to send (example: `Hello World!`)
+
+The **Channel/User Name** and **Message** can either be given as static values, or as FEEL expressions. FEEL expressions can be used to [access process variables or create the value dynamically](https://docs.camunda.io/docs/components/concepts/expressions/). This can be handy if for example a process variables is used to store the relevant channel or if the message shall be composed dynamically:
```

Comment:
Minor comment, possible to update the text to the following?
"The **Channel/User Name** and **Message** can either be given static values, or FEEL expressions. FEEL expressions can be used to [access process variables or dynamically create values](https://docs.camunda.io/docs/components/concepts/expressions/). This can be handy if for example a process variables is used to store the relevant channel or if the message needs to be composed dynamically:"

Reply:
Better indeed. Also link now relative. Updated